libvirt/domain.go

```diff
@@ -20,7 +20,7 @@ import (
 	"github.com/hashicorp/terraform/helper/resource"
 	"github.com/hashicorp/terraform/helper/schema"
 	libvirt "github.com/libvirt/libvirt-go"
-	"github.com/libvirt/libvirt-go-xml"
+	libvirtxml "github.com/libvirt/libvirt-go-xml"
```

Comment:
I don't know why is this line above changed :/ I didn't change such thing in this file intentionally, maybe the IDE changed?

Reply:
just bring it back as is master